content/connect/index.textile

```diff
@@ -126,9 +126,7 @@ An active connection ID is guaranteed to be unique in the Ably system whilst it
 
 h3(#metachannels). Connection metachannels
 
-Metachannels are a namespace of channels beginning with the [meta] qualifier, separating them from regular channels. For connections there is a specific @[meta]connection.lifecycle@ channel that publishes messages about the lifecycle of realtime connections. The connection lifecycle consists of a number of "connection states":/connect/states#available-connection-states that can be observed and interacted with using methods available on the connection object.
-
-Learn more about "metachannels here":/LINK-ME.
+"Metachannels":/metadata-stats/metadata/subscribe are a namespace of channels beginning with the [meta] qualifier, separating them from regular channels. For connections there is a specific @[meta]connection.lifecycle@ channel that publishes messages about the lifecycle of realtime connections. The connection lifecycle consists of a number of "connection states":/connect/states#available-connection-states that can be observed and interacted with using methods available on the connection object.
```

Comment:
I'd suggest "distinguishing them from regular channels" as they're not separate AFAIK.

Reply:
Good spot - updated!